snappass/main.py

```diff
@@ -25,6 +26,13 @@
 app.config.update(
     dict(STATIC_URL=os.environ.get('STATIC_URL', 'static')))
 
+# Set up Babel
+def get_locale():
```

Comment:
also need another empty line here:
```
  flake8: commands[1]> flake8
  ./snappass/main.py:30:1: E302 expected 2 blank lines, found 1
  def get_locale():
  ^
  flake8: exit 1 (0.57 seconds) /home/runner/work/snappass/snappass> flake8 pid=1916
```

Reply:
thanx for your patience; i'm learning a lot.